src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
+
+        [Option(
+            'p',
+            "project-uri",
+            HelpText = "The absolute URI of the project under which the work items are to be filed.",
+            Required = true)]
+        public string ProjectUriString { get; internal set; }
+
+        // For an explanation of the difference between ProjectUriString and ProjectUri, see FileWorkItemsCommand.ValidateOptions.
+        public Uri ProjectUri { get; internal set; }
+
+        [Option(
+            'f',
+            "filtering-strategy",
+            HelpText = "Specifies the strategy for selecting which results to file. Must be one of 'new' and 'all'.",
+            Required = true)]
+        public string FilteringStrategy { get; internal set; }
+
+        [Option(
+            'g',
+            "grouping-strategy",
```

Comment:
>grouping-strategy [](start = 13, length = 17)

I prefer 'refactoring' as a term here and elsewhere because we will, in fact, be refactoring a log file (potentially) into many others. That's a bit more of an extreme operation than grouping (which suggests reorganzing a set to me)

Reply:
I'm gonna push back on that. "Refactoring" has a very specific meaning: a code change made with the intent of improving the maintainability or extensibility of a code base. I don't see it here. I think grouping is just fine.

---
In reply to: [306575297](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306575297) [](ancestors = 306575297)

Reply:
(In an email to the SARIF TC this morning I used the word "refactoring" in precisely this sense: I suggested pulling the elements of the SARIF format that are common to static and dynamic analysis into a separate base format, the idea being to ease maintenance by allowing bugs in the common subset to be fixed in exactly one place.)

---
In reply to: [306576671](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306576671) [](ancestors = 306576671,306575297)